catkit2/services/flir_camera/flir_camera.py

```diff
@@ -209,17 +210,18 @@ def acquisition_loop(self):
                         break
                     raise
 
-                try:
-                    if image_result.IsIncomplete():
-                        continue
+                with trace_interval('processing frame'):
```

Comment:
Because that includes waiting for the camera frame itself, which isn't processing it. I'm starting the trace right after we retrieved the camera frame.